jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -284,6 +302,29 @@ private void checkNonSunProxyMemberAccess(SecurityManager security, ClassLoader
 	}
 }
 
+private long getFieldOffset(String fieldName) {
+	long fieldOffset;
+	try {
+		Field field = Class.class.getDeclaredField(fieldName);
+		fieldOffset = getUnsafe().objectFieldOffset(field);
```

Comment:
This can be `return getUnsafe().objectFieldOffset(field);`.  The local var isn't required